linera-views/benches/stores.rs

```diff
@@ -101,7 +101,7 @@ fn bench_contains_key(criterion: &mut Criterion) {
         bencher
             .to_async(Runtime::new().expect("Failed to create Tokio runtime"))
             .iter_custom(|iterations| async move {
-                let store = create_dynamo_db_test_store().await;
+                let store = DynamoDbStore::new_benchmark_store().await.unwrap();
```

Comment:
We should be able to massively deduplicate code using the new trait methods.

Reply:
Yes, we could deduplicate code with the new trait method. I agree. However, I prefer to do these transformations in stages.